plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -204,11 +214,12 @@ function exportIssues($issueIds, $context, $user) {
 	 * @param $importXml string XML contents to import
 	 * @param $context Context
 	 * @param $user User
+	 * @param $filter string Filter to be used
```

Comment:
Ah, yes, you're right -- I misread the calling code as fetching the filter object and passing it in. Never mind!